bundles/statistics/statsgrid2016/instance.js

```diff
@@ -137,10 +142,7 @@ Oskari.clazz.define(
         _addIndicatorsTabToPersonalData: function (sandbox) {
             var reqBuilder = Oskari.requestBuilder('PersonalData.AddTabRequest');
             if (typeof reqBuilder === 'function') {
-                var tab = Oskari.clazz.create('Oskari.statistics.statsgrid.MyIndicatorsTab', this);
```

Comment:
Restore the old MyIndicatorsTab and make it ReactDOM.render() the same list in the tab content that the new `MyIndicatorsTab.jsx` uses so we can have a bit of backwards compatibility with personaldata and mydata bundles:

        <MyIndicatorsList
            controller={controller}
            data={state.data}
        />